Firestore/Source/Core/FSTFirestoreClient.h

```diff
@@ -130,6 +130,8 @@ NS_ASSUME_NONNULL_BEGIN
 /** For testing only. */
 - (const std::shared_ptr<util::AsyncQueue> &)workerQueue;
 
+- (const std::atomic<bool> &)isShutdown;
```

Comment:
It seems unlikely that the caller specifically needs to access this value as an atomic, so you can probably just make the return type `bool`. This keeps the implementation details of how we're synchronizing around this value private.